### DISCUSSION TITLE
Accountability / Revision History / Tracking

### DISCUSSION THREAD

**Discussion by MentalGear** [spaceuptech/space-cloud#1638] [Q&A]
Thank you for this helpful backend.

To make it really production ready for me though, I was wondering if there are modules for Account Change Tracking (Accountability) and a revision history.

Also, are Graph DB as well supported (e.g. Surreal DB) ?

**Comment by YourTechBud** [comment]
GraphDBs are not supported.

Most users use git as a source of truth to store SC config. That can serve as your audit trail
